test/NuGet.Tests.Apex/NuGet.Tests.Apex/Apex/NuGetUIProjectTestExtension.cs

```diff
@@ -62,6 +62,23 @@ public void AssertInstalledPackageNotDeprecated()
             DeprecatedPackageResult.Should().BeFalse();
         }
 
+        public void AssertPackageTransitive(string testPackageName, string transitivePackageName)
```

Comment:
naming: This is assert more than just the transitive package.

Reply:
We updated the naming when separating this method according to the next comment.